doc/tutorials/features2d/feature_description/feature_description.markdown

```diff
@@ -49,13 +49,13 @@ int main( int argc, char** argv )
   int minHessian = 400;
 
   Ptr<SURF> detector = SURF::create();
-  detector->setMinHessian(minHessian);
+  detector->setHessianThreshold(minHessian);
```

Comment:
Maybe good idea to change the name of the parameter from minHessian to threshHessian?